Build/NuGet/Microsoft.ChakraCore.Shared.props

```diff
@@ -1,38 +1,33 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition=" (Exists('packages.config') Or Exists('packages.$(MSBuildProjectName).config')) And '$(Platform)' == 'AnyCPU'">
-    <Content Include="$(MSBuildThisFileDirectory)..\runtimes\win7-x86\native\*">
+  <ItemGroup Condition=" '$(TargetFramework)' == '' Or '$(TargetFramework.TrimEnd(`0123456789`))' == 'net' ">
```

Comment:
What does TargetFramework.TrimEnd(\`0123456789\`) do in this context?

Reply:
It removes a version number of .NET Framework (for example, `net40` or `net461`). This dismisses a target frameworks based on .NET Core (for example, `netcoreapp3.1` or `net5.0`).

This code is taken from the [official documentation](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2019).